packages/@aws-cdk-testing/framework-integ/test/aws-ses/test/integ.configuration-set-vdmoptions.ts

```diff
@@ -0,0 +1,40 @@
+import { App, Stack, StackProps } from 'aws-cdk-lib';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { Construct } from 'constructs';
+import * as ses from 'aws-cdk-lib/aws-ses';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    new ses.ConfigurationSet(this, 'VdmOptionsEnabled', {
+      vdmOptions: {
+        engagementMetrics: true,
+        optimizedSharedDelivery: true,
+      },
+    });
+
+    new ses.ConfigurationSet(this, 'VdmOptionsDisabled', {
+      vdmOptions: {
+        engagementMetrics: false,
+        optimizedSharedDelivery: false,
+      },
+    });
+
+    new ses.ConfigurationSet(this, 'VdmOptionsNotConfigured', {
+    });
+
+    new ses.VdmAttributes(this, 'VdmAccountLevelSettings', {
+      engagementMetrics: true,
+      optimizedSharedDelivery: true,
+    });
```

Comment:
So there are the account-wide settings? I would indicate the construct name in the README and the TSDoc if that's the case

Reply:
As per the other comments, the README and docs have been updated.